src/wallet/wallet.cpp

```diff
@@ -1051,14 +1051,13 @@ bool CWalletTx::WriteToDisk(CWalletDB *pwalletdb)
     return pwalletdb->WriteTx(GetHash(), *this);
 }
 
-bool CWallet::WitnessBucketCommitment(uint256 &commitment,
-                                      libzcash::MerklePath &path,
+void CWallet::WitnessBucketCommitment(std::vector<boost::optional<uint256>> commitments,
+                                      std::vector<boost::optional<ZCIncrementalWitness>>& witnesses,
                                       uint256 &final_anchor)
```

Comment:
Hm, you would never, so this API is kind of funky. It's meant to optimize away comparisons once we've witnessed some or all of the commitments, but that isn't very helpful anyway, so I'll just remove it.